main.tf

```diff
@@ -42,6 +45,13 @@ locals {
       "Name" = var.append_schedule_postfix ? "${replace(index, "_", "-")}-schedule" : index
     })
   ])
+  eventbridge_pipes = flatten([
+    for index, pipe in var.pipes :
+    merge(pipe, {
+      "name" = index
+      "Name" = var.append_pipe_postfix ? "${replace(index, "_", "-")}-pipe" : index
```

Comment:
What is the postifix and `-pipe`; is that a requirement by AWS?

Reply:
The prefix/postfix is not the requirement by AWS, but this is how it is handled in this module already. We don't handle it this way in other serverless terraform-aws-modules.

Postfix + `Name`/`name` + tagging must be updated for all resources in a separate future release (with breaking changes).